### ISSUE TITLE
Redoc use Ukranian language

### ISSUE THREAD

**Issue by lyrixx** [api-platform/core#4394]
**API Platform version(s) affected**: 2.6

**Description**  
https://demo.api-platform.com/docs?ui=re_doc#tag/Book
=> 
```
"hydra:view": {
"@id": "../словник",
"@type": "string",
"hydra:first": "../словник",
"hydra:last": "../словник",
"hydra:previous": "../словник",
"hydra:next": "../словник"
},
```

It would be better to use plain English by default

**Comment by norkunas** [comment]
#4354 related

**Comment by lyrixx** [comment]
I "fixed" it in #4395
